### DISCUSSION TITLE
Rmarkdown format of tabset not rendered using quarto render

### DISCUSSION THREAD

**Discussion by rhshah** [quarto-dev/quarto-cli#2717] [Q&A]
Currently we have Rmarkdown files that use the Rmarkdown format of tabset https://bookdown.org/yihui/rmarkdown-cookbook/html-tabs.html and these don't render as tabs when using quarto to render them. Is this the expected behavior?

**Comment by dragonstyle** [comment]
This is expected- we have an alternate syntax for tabs (that is similar, but not identical):

https://quarto.org/docs/interactive/layout.html#tabset-panel
